frontend/public/co-fetch.js

```diff
@@ -164,7 +164,12 @@ export const coFetchJSON = (url, method = 'GET', options = {}, timeout) => {
 };
 
 export const coFetchText = (url, options = {}, timeout) => {
-  return coFetchCommon(url, 'GET', options, timeout);
+  return coFetchCommon(url, 'GET', options, timeout).then((response) => {
```

Comment:
```suggestion
  return coFetchCommon(url, 'GET', options, timeout).then((response): string => {
```

Reply:
@andrewballantyne I dint change this because this is a co-fetch.js File, i can change it to ts file but all other methods are not typed here. WDYT?

Reply:
Oh hahaha, sorry... on autopilot. Forget we have JS files. Nvm, you are good, ignore this change.